package.json

```diff
@@ -91,7 +91,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/kenwheeler/nuka-carousel.git"
+    "url": "https://github.com/carlomnop/nuka-carousel.git"
```

Comment:
why were these changed?

Reply:
My bad, I forgot to update. I started working from the fork of the repo https://github.com/carlomnop/nuka-carousel/tree/master from which these changes derive.